### PR TITLE
add unibox v1.12.0 with git auto-update

### PR DESCRIPTION
**Pull request by maruilian11**
**git repo url:** https://github.com/ddsky/unibox
@PeterDaveHello  could u help me check this PR for #7432 ? thank you!

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ ] Has valid tags for each versions (for git auto-update) https://github.com/ddsky/unibox/issues/4
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/unibox/package.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/ddsky/unibox.git"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/ddsky/unibox.git",
+    "basePath": "",
+    "files": [
+      "(css|js)+/unibox*.+(js|css|map)"
```
Comment (PeterDaveHello) [inline ajax/libs/unibox/package.json]: What does `(css|js)+` mean?

Comment (maruilian11) [inline ajax/libs/unibox/package.json]: @PeterDaveHello `css` and `js` folder

Comment (PeterDaveHello) [inline ajax/libs/unibox/package.json]: So you defined or invented your own syntax? I can't see it in https://github.com/isaacs/node-glob

Comment (maruilian11) [inline ajax/libs/unibox/package.json]: what's the relationship between `ddsky/unibox` and `isaacs/node-glob`

Comment (maruilian11) [inline ajax/libs/unibox/package.json]: sorry, i mistook your meaning entirely, i have updated the commit.

File: ajax/libs/unibox/package.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    "type": "git",
+    "url": "https://github.com/ddsky/unibox.git"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/ddsky/unibox.git",
+    "basePath": "",
+    "files": [
+      "css/unibox*.+(css|map)",
+      "js/unibox*.+(js|map)"
```
Comment (PeterDaveHello) [inline ajax/libs/unibox/package.json]: `"+(js|css)/unibox*.+(js|css|map)"`

**Comment by maruilian11** [comment]
@PeterDaveHello updated

**Comment by PeterDaveHello** [comment]
Know what your problem is now?

**Comment by PeterDaveHello** [comment]
Please talk about the auto-update config in PR title.

**Comment by maruilian11** [comment]
yes, thanks.

**Pull request merged**